tests/e2e/e2e_controller_restart_test.go

```diff
@@ -113,13 +113,13 @@ func testHTTPTrafficWithControllerRestart() {
 			result := Td.HTTPRequest(clientToServer)
 
 			if result.Err != nil || result.StatusCode != 200 {
-				Td.T.Logf("> (%s) HTTP Req failed %d %v",
-					srcToDestStr, result.StatusCode, result.Err)
+				Td.T.Logf("%s > (%s) HTTP Req failed %d %v",
```

Comment:
One thing I'd do (eventually) is add a bit more context around these %s.
When I saw the greater-than sign - I thought we are comparing something - but this seems to be a visual separator?
I'd make this human readable with something like: `At %+v HTTP request %s failed with status code %d and error: %v`
I think this would make the code easier to read and the log message when one runs into it.

Reply:
Noted. The intention  long term would probably be to add this details in the logger itself.